src/en/rokaricomics/src/eu/kanade/tachiyomi/extension/en/rokaricomics/RokariComics.kt

```diff
@@ -1,14 +1,120 @@
 package eu.kanade.tachiyomi.extension.en.rokaricomics
 
 import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
 import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
 
 class RokariComics : MangaThemesia(
     "RokariComics",
     "https://rokaricomics.com",
     "en",
 ) {
-    override fun chapterListSelector() = "#chapterlist li:has(div.chbox):has(div.eph-num):has(a[href])"
+    // Popular - Use homepage "Popular Today" section (first page only, no pagination)
+    override fun popularMangaRequest(page: Int): Request {
+        return if (page == 1) {
+            GET(baseUrl, headers)
+        } else {
+            // Return empty page for page > 1 since Popular Today is not paginated
+            GET("$baseUrl/page/0/", headers)
+        }
```

Comment:
```suggestion
        GET(baseUrl, headers)
```
you already pass `hasNextPage=false`, so no need to provide alternative code path